### PR TITLE
gh-release 2.0.0 (new formula)

### PR DESCRIPTION
**Pull request by ekristen**
Utility for automating Github releases with file uploads

### CODE REVIEW

File: Library/Formula/gh-release.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+class GhRelease < Formula
+  desc "Utility for automating GitHub releases with file uploads"
+  homepage "https://github.com/progrium/gh-release"
+  url "https://github.com/progrium/gh-release/releases/download/v2.2.0/gh-release_2.2.0_darwin_x86_64.tgz"
```
Comment (mikemcquaid) [inline Library/Formula/gh-release.rb]: This needs to be built from source.

File: Library/Formula/gh-release.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  version "2.2.0"
+  sha256 "b272576d9fd6449caa30e64e82f683a0bbc240eadf3be42fad4d8d2c7bd86eef"
+
+  def install
+    bin.install "gh-release"
+  end
+
+  test do
+    system "#{bin}/gh-release", "--help"
```
Comment (mikemcquaid) [inline Library/Formula/gh-release.rb]: Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!

Comment (ekristen) [inline Library/Formula/gh-release.rb]: Nope, it all requires authentication to GitHub to do any of its functions.

File: Library/Formula/gh-release.rb (hunk 1 above)
Comment (ekristen) [inline Library/Formula/gh-release.rb]: Ok.

**Comment by DomT4** [comment]
Ping on Mike's comments?

**Comment by bfontaine** [comment]
Closing due to no response. Feel free to open another PR that build from source. Thanks!

**Pull request closed without merging**